### PR TITLE
:sparkles: Add `values` to ConfigurationResponse

### PR DESCRIPTION
**Pull request by yoerriwalstra**
**Changes**
- add `Value` class
- add `ValueCollection` class
- include `values` in ConfigurationResponse
- update tests

### CODE REVIEW

File: tests/Configuration/ConfigurationResponseTest.php [hunk 1]
```diff
@@ -45,6 +47,41 @@ public function test_it_creates_a_configuration_response_with_minimal_data(): vo
...
+    {
+        $form = Mockery::mock(Form::class, [
+            'toArray'     => [],
+            'getRequired' => [],
+        ]);
+
+        $faker = Factory::create();
+
+        $name = $faker->word;
+        $value = $faker->word;
```
Comment (yoan-myparcel) [inline tests/Configuration/ConfigurationResponseTest.php:60]: faker will issue warning for usage of the magic props:
```suggestion
        $name = $faker->word();
        $value = $faker->word();
```

File: tests/Configuration/ConfigurationResponseTest.php [hunk 2]
```diff
@@ -94,7 +131,12 @@ enum: ['1', '2', '3'],
             $select,
         );
 
-        $configuration = new ConfigurationResponse($form);
+        $name = $faker->word;
+        $value = $faker->word;
```
Comment (yoan-myparcel) [inline tests/Configuration/ConfigurationResponseTest.php:135]:
```suggestion
        $name = $faker->word();
        $value = $faker->word();
```

File: tests/Configuration/Values/ValueTest.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+use PHPUnit\Framework\TestCase;
+
+class ValueTest extends TestCase
+{
+    public function testValue(): void
+    {
+        $faker = Factory::create();
+
+        $name = $faker->word;
+        $value = $faker->word;
```
Comment (yoan-myparcel) [inline tests/Configuration/Values/ValueTest.php:17]:
```suggestion
        $name = $faker->word();
        $value = $faker->word();
```

File: tests/Configuration/Values/ValueTest.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+declare(strict_types=1);
+ namespace Tests\Configuration\Values;
+
+use Faker\Factory;
+use MyParcelCom\Integration\Configuration\Values\Value;
+use PHPUnit\Framework\TestCase;
+
+class ValueTest extends TestCase
+{
+    public function testValue(): void
```
Comment (yoan-myparcel) [inline tests/Configuration/Values/ValueTest.php:12]:
```suggestion
    public function test_value(): void
```

File: tests/Configuration/ConfigurationResponseTest.php (hunk 1 above)
Comment (yoerriwalstra) [reply tests/Configuration/ConfigurationResponseTest.php]: Give warnings where? I don't see them in my IDE, nor terminal, nor pipeline

**Review by yoan-myparcel — APPROVED** [review]

File: tests/Configuration/ConfigurationResponseTest.php (hunk 1 above)
Comment (yoerriwalstra) [reply tests/Configuration/ConfigurationResponseTest.php]: ohhh, must be related to that flag then that I never noticed this before!

**Pull request merged**